### PR TITLE
Update examples

### PR DESCRIPTION
**Pull request by lmangani**
Thanks @ramazanpolat for the suggestion and for opening issue #1

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -292,7 +294,7 @@ <h2 class="home-text44">
                 </span>
                 <span class="home-answer3 textarea">
                   <span class="home-text51">
-                    :) INSERT INTO FUNCTION url(getSetting(&apos;</span><span style="color: orange">custom_url</span><span>&apos;), JSONEachRow, &apos;key String, value UInt64&apos;) VALUES
+                    :) INSERT INTO FUNCTION url(</span><span style="color: orange">{url:String}</span><span>), JSONEachRow, &apos;key String, value UInt64&apos;) VALUES
```
Comment (ramazanpolat) [inline index.html]: There is an extra closing paren
`:) INSERT INTO FUNCTION url(</span><span style="color: orange">{url:String}</span><span>), JSONEachRow, &apos;key String, value UInt64&apos;) VALUES`
needs to be changed to 
`:) INSERT INTO FUNCTION url(</span><span style="color: orange">{url:String}</span><span>, JSONEachRow, &apos;key String, value UInt64&apos;) VALUES`

File: index.html [hunk 2]
```diff
@@ -310,7 +312,7 @@ <h2 class="home-text44">
                 </span>
                 <span class="home-answer5 textarea">
                   <span class="home-text55">
-                    :) SELECT * FROM url(getSetting(&apos;</span><span style="color: orange">custom_url</span><span>&apos;), JSONEachRow)
+                    :) SELECT * FROM url(</span><span style="color: orange">{url:String}</span><span>), JSONEachRow)
```
Comment (ramazanpolat) [inline index.html]: Same here. Needs to be:
`:) SELECT * FROM url(</span><span style="color: orange">{url:String}</span><span>, JSONEachRow)`

**Review by ramazanpolat — CHANGES_REQUESTED** [review]

**Comment by ramazanpolat** [comment]
Also, [line 385](https://github.com/metrico/urleng/blob/1d87aae99f06d99319074fe8124d19fd561e4afc/index.html#L385) needs to be changed.

**Comment by lmangani** [comment]
Thanks @ramazanpolat i'm doing this from a mobile device with the web editor and its probably a huge mistake :)
EDIT: Feel free to modify the PR and take credit for this fix!

**Pull request merged**